src/nft/css/cssObjectFromTheme.ts

```diff
@@ -0,0 +1,30 @@
+import { assignInlineVars } from '@vanilla-extract/dynamic'
+
+import { Theme, themeVars } from './sprinkles.css'
+
+const resolveTheme = (theme: Theme | (() => Theme)) => (typeof theme === 'function' ? theme() : theme)
+
+export function cssObjectFromTheme(
+  theme: Theme | (() => Theme),
+  { extends: baseTheme }: { extends?: Theme | (() => Theme) } = {}
+) {
+  const resolvedThemeVars = {
+    // We use an object spread here to ensure it's a plain object since vanilla-extract's
```

Comment:
The functionality of this fn has expanded since when this comment was written. Moving away from the custom theme to cssObject function would be a decent refactor that I believe is outside of the scope of this PR. We can circle back on this when we have made a long term decision on styling architecture? I can remove the comment for now to avoid confusion.